feedback/w07-feedback-feedback.md

```diff
@@ -0,0 +1,66 @@
+# Feedback on Week 6 Feedback
+
+## Whole-Team Action Items
+- Supporting instructors will now deliver all homework assignments to ensure all bases are covered with respect to what was delivered in class.
+- Not slacking during classrooms / lessons / outcomes programming unless it’s urgent.
```

Comment:
SUGGESTED EDIT: Not Slacking students during lessons/programming unless it's urgent and/or directly related to the activity at hand.